grpc/src/test/java/com/linecorp/armeria/internal/common/grpc/GrpcStatusTest.java

```diff
@@ -58,4 +59,10 @@ void failFastExceptionToUnavailableCode() {
                              .getCode())
                 .isEqualTo(Status.Code.UNAVAILABLE);
     }
+
+    @Test
+    void invalidProtocolBufferExceptionToInvalidArgumentCode() {
+        assertThat(GrpcStatus.fromThrowable(new InvalidProtocolBufferException("Failed to parse message")).getCode())
+                .isEqualTo(Status.Code.INVALID_ARGUMENT);
+    }
```

Comment:
Could we also add an integration test that sends invalid data described in the issue?

Reply:
Okay, I'll